presto-docs/src/main/sphinx/release/release-0.216.rst

```diff
@@ -0,0 +1,65 @@
+=============
+Release 0.216
+=============
+
+General Changes
+---------------
+
+* Fix incorrect query result for queries involving timestamp, when the time zone database
+  embedded in the version of Joda-Time that Presto depends on does not match the time zone
+  database used by the JVM that Presto runs on.
+* Fix count(*) aggregation on empty relation when ``optimize_mixed_distinct_aggregation`` is enabled.
+* Improve performance of table scan on structured types by removing unnecessary memory accounting.
+* Add `reduce_agg` aggregate function.
+* Add :func:`millisecond` function (:issue:`#9524`).
+* Add an optimizer rule to push filters on window function partition symbols under the window
+  operator.
+* Remove ``SHOW STATS ON`` syntax.
+* Restrict ``WHERE`` clause in ``SHOW STATS`` to filters that can be pushed down to the connector.
+* Remove ``node_id`` column from ``system.runtime.queries`` table.
+* Return final results to clients immediately for failed queries.
+
+Web UI
+------
+
+* Fix the rendering of live plan view for queries involving index joins (:issue:`12054`).
+
+Hive Changes
+------------
+
+* Fix accounting of time spent reading Parquet data.
+* Fix a corner case where ORC writer fails with "integer overflow" when writing
+  highly compressible data using dictionary encoding (#11930).
+* Queries reading Parquet files will now fail by default if statistics
+  in those Parquet files are corrupt (e.g., min > max).
+  To disable this behavior, set the configuration property ``hive.parquet.fail-on-corrupted-statistics``
+  or session property ``parquet_fail_with_corrupted_statistics`` to false.
+
+Kudu Changes
+------------
+
+* Update client to 1.8.0.
+* Add table property ``number_of_replicas`` to ``SHOW CREATE TABLE`` output.
+
+Cassandra Changes
+-----------------
+
+* Add ``cassandra.splits-per-node`` and ``cassandra.protocol-version`` configuration properties
+  to allow connecting to Cassandra servers older than 2.1.5.
+
+MySQL, PostgreSQL, Redshift, and SQL Server Changes
+---------------------------------------------------
+
+* Add support for predicate pushdown for columns of ``char(x)`` type.
+
+Verifier
+--------
+
+* Add ``run-teardown-on-result-mismatch`` configuration property. When set to false,
+  temporary tables will not be dropped after checksum failure to facilitate debugging.
+
+SPI
```

Comment:
SPI Changes

Reply:
@electrum David, would you update https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines with the new guidelines?

Reply:
Done